lm_eval/models/mlc_llm.py

```diff
@@ -309,7 +310,7 @@ def prepare_msg(self, request):
         request_args = request[1]
         until = request_args["until"]
 
-        return self.create_chat_completion_payload(prompt, until)
+        return self.create_chat_completion_payload(prompt)
```

Comment:
Looks like it is not bag, it is feature of the test. It uses until to stop token generation earlier when it stops by it-self. Ok, it badly influences on result, but it is main pipeline of lm-evaluation-harness